sysbuild/Kconfig.netcore

```diff
@@ -10,14 +10,15 @@ config EXTERNAL_CONFIGURED_NETCORE
 
 config SUPPORT_NETCORE
 	bool
-	default y if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp_ns")
+	default y if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp_ns" || $(BOARD) = "nrf54h20dk_nrf54h20_cpuapp")
 
 config NETCORE_REMOTE_BOARD_NAME
 	string
 	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
 	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns"
 	default "thingy53_nrf5340_cpunet"  if $(BOARD) = "thingy53_nrf5340_cpuapp"
 	default "thingy53_nrf5340_cpunet"  if $(BOARD) = "thingy53_nrf5340_cpuapp_ns"
+	default "nrf54h20dk_nrf54h20_cpurad"  if $(BOARD) = "nrf54h20dk_nrf54h20_cpuapp"
```

Comment:
This needs to wait for #14529 to get in first, right? Otherwise, you'll need to use `nrf54h20pdk*`.

Reply:
yes that is correct. I don't see a point to make this for PDK.